ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -457,4 +489,39 @@ export class IndexerClient {
     return (await this.getCollectionData(creatorAddress, collectionName, extraArgs)).current_collections_v2[0]
       .collection_id;
   }
+
+  /**
+   * Queries for all collections that an account has tokens for.
+   *
+   * @param ownerAddress the account address that owns the tokens
+   * @returns the collection address
+   */
+  async getCollectionsWithOwnedTokens(
+    ownerAddress: MaybeHexString,
+    extraArgs?: {
+      tokenStandard?: TokenStandard;
+      options?: PaginationArgs;
+    },
+  ): Promise<GetCollectionsWithOwnedTokensQuery> {
```

Comment:
The comment about what this returns and the return type here don't match, could you update that?